### DISCUSSION TITLE
help me set up lapce for web development

### DISCUSSION THREAD

**Discussion by theArianit** [lapce/lapce#2810] [General]
Hello guys. I came across Lapce while looking for vsCode alternatives (zed editor was in my list for some time, but mac only made me feel dissapointed on the project). I know that this project is new, and has many issues and maybe shouldn't be used for production, but I want to use it for web development, since not much is required and not much can go wrong. 
So, how can I get started with Lapce for web dev? I am a complete noob in this regards, so I need instructions of how to make language servers and such to work.
Thanks in advance!
p.s. the flatpak needs an update ;)

**Comment by theArianit** [comment]
setup for svelte/vue development would be nice

**Comment by Snowiiii** [comment]
Hey, Great to hear you try out Lapce!, So at the beggining you should know that this Project is really not old yet and their are not many Plugins or Features yet, But lets hope this will change :D. Okay here are my Recommendations:

- [Vue](https://plugins.lapce.dev/plugins/xiaoxin-sky/lapce-vue) (For Vue stuff)
- [Rome](https://plugins.lapce.dev/plugins/xiaoxin-sky/lapce-rome) (Should Cover JS/TS and HTML/CSS, Has Markdown support as well)
- [PHP](https://plugins.lapce.dev/plugins/dajoha/lapce-php-intelephense) (Many Web devs use PHP idk about you)

When you don't like Rome you can get Stuff seprate as well
- [HTML](https://plugins.lapce.dev/plugins/CroxxN/lapce-html) 
-[TS/JS](https://plugins.lapce.dev/plugins/panekj/lapce-typescript)

Okay thats it i guess, So you said your coming from VSCode and when you like the VSCode Theme you can get it in Lapce as well
- [VSCode Theme](https://plugins.lapce.dev/plugins/panekj/vscode-themes)

**Reply by theArianit** [reply]
Hi, thanks for helping out. Do I have to configure anything after installing these extensions?

**Comment by Snowiiii** [comment]
Ye like the most Plugins already in their readme says you often need to install an LSP using npm and then may export the path var.The only thing you may need to change is the theme in the settings
